### PR TITLE
Fixed unit tests

### PR DESCRIPTION
**Pull request by JanMaartenvanDoorn**
Hello, here a pull-request that should fix all unit test issues. I removed the old unit tests that were testing code which is removed and adapted unit tests that were testing modified code. There is still one unit test failing: the test "test_something" which I think was left there intentionally. Monday I will have a look if I can expand coverage on the training pipeline a little bit.

### CODE REVIEW

File: openstf/pipeline/optimize_hyperparameters.py [hunk 1]
```diff
@@ -1,166 +0,0 @@
-# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
```
Comment (touronaug) [inline openstf/pipeline/optimize_hyperparameters.py:1]: Why was this file deleted?

Comment (JanMaartenvanDoorn) [reply openstf/pipeline/optimize_hyperparameters.py:1]: This hyperparameter optimization still uses the old models so I decided to delete it on this branch. Optimizing the new sklearn compatible models probably has to be implemented differently. I see there is currently no issue for this but I will make one.
